core/templates/pages/teach-page/teach-page.component.html

```diff
@@ -1,137 +1,943 @@
+          <span (click)="decrementDisplayedTestimonialId()" class="material-icons">
+            arrow_back_ios
+          </span>
+          <span (click)="incrementDisplayedTestimonialId()" class="material-icons">
+            arrow_forward_ios
+          </span>
         </div>
       </div>
     </div>
   </div>
-  <div class="oppia-footer-padding-below-banner">
+
+  <div class="oppia-teach-section-six">
+    <div *ngIf="!isWindowNarrow">
+      <h2 [innerHTML]="'I18N_TEACH_PAGE_SIX_TITLE' | translate"></h2>
+      <!--<div class="oppia-teach-button-container-button-one">
+        <form action="/teach">
+          <button type="submit" (click)="onClickGuideParentsButton()"
+                  class="btn oppia-teach-bottom-button oppia-teach-button-browse oppia-teach-browse-button-position oppia-transition-200"
+                  [innerHTML]="'I18N_ACTION_GUIDE_FOR_TEACHERS' | translate">
+          </button>
+        </form>
+      </div>
+      <div class="oppia-teach-button-container-button-one">
+        <form action="/teach">
+          <button type="submit" (click)="onClickTipforParentsButton()"
+                  class="btn oppia-teach-bottom-button oppia-teach-button-browse oppia-teach-browse-button-position oppia-transition-200"
+                  [innerHTML]="'I18N_ACTION_TIPS_FOR_PARENTS' | translate">
+          </button>
+        </form>
+      </div> -->
+      <div class="oppia-teach-button-container-button-one">
+        <form action="classroomUrl">
+          <button type="submit" (click)="onClickExploreLessonsButton()"
+                  class="btn oppia-teach-bottom-button oppia-teach-button-browse oppia-teach-browse-button-position oppia-transition-200"
+                  [innerHTML]="'I18N_ACTION_EXPLORE_LESSONS' | translate">
+          </button>
+        </form>
+      </div>
+    </div>
+
+    <div *ngIf="isWindowNarrow">
+      <h2 [innerHTML]="'I18N_TEACH_PAGE_SIX_TITLE' | translate"></h2>
+      <!--Removing these 2 buttons for now. Will add them back after Tips for parents and guide for teachers  pages are ready
+        <div class="oppia-teach-button-container-button-one">
+        <form action="/teach">
+          <button type="submit" (click)="onClickGuideParentsButton()"
+                  class="btn oppia-teach-bottom-button mobile oppia-teach-button-browse oppia-teach-browse-button-position oppia-transition-200"
+                  [innerHTML]="'I18N_ACTION_GUIDE_FOR_TEACHERS' | translate">
+          </button>
+        </form>
+      </div>
+      <div class="oppia-teach-button-container-button-one">
+        <form action="/teach">
+          <button type="submit" (click)="onClickTipforParentsButton()"
+                  class="btn oppia-teach-bottom-button mobile oppia-teach-button-browse oppia-teach-browse-button-position oppia-transition-200"
+                  [innerHTML]="'I18N_ACTION_TIPS_FOR_PARENTS' | translate">
+          </button>
+        </form>
+      </div> -->
```

Comment:
ditto

Reply:
It was discussed in the PR thread and an explanation for the same is already added above the code. These buttons are temporarily removed because the pages were not ready. As soon as they are ready we will get this uncommented.